pages/blog/_slug.vue

```diff
@@ -17,14 +17,30 @@
 
 <script>
 export default {
-  async asyncData ({ $content, params, error }) {
-    const [post] = await $content('blog', { deep: true })
-      .where({ dir: `/blog/${params.slug}` })
+  async asyncData ({ $content, params, error, app }) {
+    const currentLocale = app.i18n.locale
+    const isEnglish = currentLocale === 'en'
```

Comment:
Use environment variable and maybe even rename the variable:
```suggestion
    const isDefault = currentLocale === process.env.DEFAULT_LANG
```

Reply:
Done.